client/containers/Pub/SpubHeader/InstructionsTab.tsx

```diff
@@ -8,13 +9,15 @@ import SpubHeaderTab from './SpubHeaderTab';
 require('./instructionsTab.scss');
 
 type Props = {
+	onBeginSubmission: () => any;
```

Comment:
This is nitpicking but I might as well say it: as I understand it, the "best" return type for callback props that don't return anything is `unknown`. This admits the same set of functions as `any`, but it forbids the component from doing willy-nilly property access like `onBeginSubmission().some.thing`.

Reply:
Right! Meant to ask how to type props-passed functions when we expect  them to be state-setters like this one is.
Unknown is peachy, but what in `[someState, setSomeState] = useState('');` what is the return type of `setSomeState`?